app/containers/App/index.js

```diff
@@ -39,37 +41,37 @@ const AppWrapper = styled.div`
   flex-direction: column;
 `;
 
-export const App = ({ handleError: errorHandler }) => (
-  <AppWrapper>
-    <Helmet
-      titleTemplate="%s - React.js Boilerplate"
-      defaultTitle="React.js Boilerplate"
-    >
-      <meta name="description" content="A React.js Boilerplate application" />
-    </Helmet>
-    <ErrorBoundary onError={errorHandler} FallbackComponent={<ErrorFallback />}>
+export function App({ location }) {
+  return (
+    <AppWrapper>
+      <Helmet
+        titleTemplate="%s - React.js Boilerplate"
+        defaultTitle="React.js Boilerplate"
+      >
+        <meta name="description" content="A React.js Boilerplate application" />
+      </Helmet>
       <Header />
-      <Switch>
+      <Switch location={location}>
```

Comment:
What is being achieved by passing the location prop to `Switch`?

Reply:
Fix page not refresh when routing change.

Reply:
:+1: